src/main/java/edu/kis/powp/command/ComplexCommand.java

```diff
@@ -0,0 +1,17 @@
+package edu.kis.powp.command;
+
+import java.util.ArrayList;
+
+public class ComplexCommand implements DriverCommand {
+	ArrayList<DriverCommand> driverCommands;
```

Comment:
List type (attribute and in the constructor) would be sufficient and would not limit the implementation.

Should be private!